src/target_intacct/employee_rate.py

```diff
@@ -28,7 +28,7 @@ def employee_rate_upload(intacct_client) -> None:
     input_value = get_input()
 
     # Convert input from dictionary to DataFrame
-    data_frame = pd.DataFrame(input_value)
+    data_frame = pd.DataFrame(input_value[0])
```

Comment:
any risk that the input_value will not be a list or tuple?

Reply:
I don't believe so but I'll double check that. There might be a better way to do it regardless